source/sink-connector/configuration-properties/metrics.txt

```diff
@@ -0,0 +1,288 @@
+
+   {+jmx-hover+} represents an individual metric as an attribute of an ``MBean``.
+   To learn more about attributes and ``MBeans``, see the
+   `Standard MBeans Tutorial <https://docs.oracle.com/javase/tutorial/jmx/mbeans/standard.html>`__
+   from Oracle.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Attribute Name
+     - Description
+
+   * - | **records**
+       | 
+     - | **Description:**
+       | The total number of Kafka records a MongoDB sink task received.
+
+   * - | **records-successful**
+       | 
+     - | **Description:**
+       | The total number of Kafka records a MongoDB sink task successfully
+         wrote to MongoDB.
+
+   * - | **records-failed**
+       | 
+     - | **Description:**
+       | The total number of Kafka records a MongoDB sink task failed to write
+         to MongoDB.
+
+   * - | **latest-kafka-time-difference-ms**
+       | 
+     - | **Description:**
+       | The number of milliseconds of the most recent time difference recorded
+         between a MongoDB sink task and Kafka. This value is calculated by
+         subtracting the current time of the connector's clock and the timestamp
+         of the last record the task received.
+
+   * - | **in-task-put**
+       | 
+     - | **Description:**
+       | The total number of times the Kafka Connect framework executed the
+         ``put()`` method of the MongoDB sink task.
+
+   * - | **in-task-put-duration**
+       | 
+     - | **Description:**
+       | The total number of milliseconds the Kafka Connect framework spent
+         executing the ``put()`` method of a MongoDB sink task.
+
+   * - | **in-task-put-duration-over-1-ms**
+       | 
+     - | **Description:**
+       | The total number of MongoDB sink task ``put()`` method executions
+         with a duration that exceeded 1 millisecond.
+
+   * - | **in-task-put-duration-over-10-ms**
+       | 
+     - | **Description:**
+       | The total number of MongoDB sink task ``put()`` method executions
+         with a duration that exceeded 10 milliseconds.
+
+   * - | **in-task-put-duration-over-100-ms**
+       | 
+     - | **Description:**
+       | The total number of MongoDB sink task ``put()`` method executions
+         with a duration that exceeded 100 milliseconds.
+
+   * - | **in-task-put-duration-over-1000-ms**
+       | 
+     - | **Description:**
+       | The total number of MongoDB sink task ``put()`` method executions
+         with a duration that exceeded 1000 milliseconds.
+
+   * - | **in-task-put-duration-over-10000-ms**
+       | 
+     - | **Description:**
+       | The total number of MongoDB sink task ``put()`` method executions
+         with a duration that exceeded 10000 milliseconds.
+
+   * - | **in-connect-framework**
+       | 
+     - | **Description:**
+       | The total number of times code in the Kafka Connect framework
+         executed after the first invocation of the ``put()`` method of the
+         MongoDB sink task.
```

Comment:
Issue:
I think in the spreadsheet, there were a few different requests to elaborate on certain items in the documentation (the sheet represents source code comment descriptions). In this specific case, Robert requested the following:

> lets make sure we document what the significance of the put() method is

Could you check on the sheet and make sure these are addressed in the documentation? Alternatively, create a ticket that includes all the asks to elaborate/link to more information in the documentation.

Reply:
Created follow up ticket to add [elaborations](https://jira.mongodb.org/browse/DOCSP-25229)'